### ISSUE TITLE
The Email and Job Title not read to screen reader

### ISSUE THREAD

**Issue by dynamic11** [CDH-Studio/UpSkill#1526]
**Describe the bug**
The job title and email should be read even though they are disabled in the contact form

![image](https://user-images.githubusercontent.com/11470442/124312449-588c2680-db3d-11eb-9588-5fdbad9a3247.png)

**Comment by dynamic11** [comment]
@MenaMachado did you get a chance to look into the wording I sent to you in the email?
